supplementary/alica_ros1/supplementary_tests/alica_supplementary_test_library/etc/plans/conditions/ConditionRepository.cnd

```diff
@@ -0,0 +1,26 @@
+{
+  "comment": "",
+  "conditions": [
+    {
+      "blackboard": [],
+      "comment": "",
+      "conditionRepository": 0,
+      "id": 295816226925111421,
+      "name": "VariableHandlingStart",
+      "relativeDirectory": "conditions",
+      "libraryName": "alica_supplementary_test_library"
```

Comment:
This is not required for each condition in the file instead, the library name should be part of the outer yaml node

Reply:
We need to be able to use conditions for other libraries right?

Reply:
I guess it depends on how we want to store the conditions in the main application's etc folder. So, each library has a cnd file with the conditions it defines & the main application can either
- have a copy of each library's (that it uses) cnd file in its folder & the engine can read all of the cnd files
- or it can have 1 cnd file that is the aggregate of all conditions that are used 
I'm not sure which is easier to do from backend's perspective

Reply:
I think the second is cleaner from design perspective.

Reply:
Yeah ok, in that case this PR is fine as it is

Reply:
However I guess if it is fine to store library conditions in the library package and not duplicate them then it's fine to take a multi file approach

Reply:
Yep go ahead